addons/mail/static/src/core/common/message.xml

```diff
@@ -11,11 +11,13 @@
                 role="group"
                 t-att-aria-label="messageTypeText"
                 t-on-click="onClick"
+                t-on-contextmenu="onContextMenu"
```

Comment:
[This is not supported by Safari](https://developer.mozilla.org/fr/docs/Web/API/Element/contextmenu_event#compatibilit%C3%A9_des_navigateurs).

If we need to support it on mobile, we need to do something for Safari

Reply:
Longpress in mobile is already supported in the code. We don't use `contextmenu` because as you point out it doesn't work. The PR is merely adding right-click in desktop.